docs/NPC_Swap_Tutorial.md

```diff
@@ -0,0 +1,224 @@
+
+You should now have the `00873434CB4F9FCD.entity.json` file and 5 folders (BORG - TEXT) in the output folder.
+
+# Modifying the Files
+The next step is to find the TEMP hash value for the outfit that is to be replaced: the default Signature Suit. All of 47's suits begin with the string `outfit_agent47`, and as mentioned earlier, they are all located in `chunk0`. Therefore, you can search for the string `outfit_agent47` using RPKG Tool's `Search RPKGs` function, and then examine the results in `chunk0`, `chunk0patch1`, or `chunk0patch2`. Many of the filenames will match up with the suit's in-game name, but many will not. Some suits have their own codenames\*, and some use the codename for the level they're from\*\*. Finally, you'll notice that many suits have both an `actor` and a `heroa` variant. The actor variants are for NPCs, while the heroa variant is specifically for 47, so you'll always want to use the heroa variants when creating NPC or outfit swaps.
+
+In this case, the default Signature Suit is fairly easy to find, as it's the second result in the chunk0 TEMP tree and matches up with the in-game name (`outfit_agent47_signaturesuit_heroa_v0`). If we look at the result in the left box, we can see it begins with `00FF8C6314EA882E.TEMP`, so the value we're looking for is `00FF8C6314EA882E`. Make a note of this hash.
+
+Alternatively, we can use HMBM47's [Hitman 3 Outfits Spreadsheet](https://docs.google.com/spreadsheets/d/e/2PACX-1vRDiyiqdRebu0Olvvkr20CDhh6ANxu7FOQZ_O-1YHFN9e6kh0WmpbwDYbfgzevSvc3fO4_4Exu1fmQH/pubhtml#), only this time, we'll access the `chunk0 47 Base Suits` section. As before, we'll see each suit's in-game name in the `Name` column, and the suit's filename in the `Outfit` column. However, we'll need the TEMP hash rather than the filename, so we can locate that in the `TEMP` column. First, if we search for `Signature Suit`, we can identify which is the default version and which is the gloved version by looking at either the `Notes` column, or the filename in the `Outfit` column. Then, we can identify the outfit's TEMP by looking at the TEMP column on the left, which is `00FF8C6314EA882E.TEMP`. Therefore, the hash we need is `00FF8C6314EA882E`
+
+In either case, once you have the TEMP hash, the next step is to open the `00873434CB4F9FCD.entity.json` file (which we previously exported) using a text editor. You'll see it starts with: `{"tempHash":"00873434CB4F9FCD"`. All we need to do is replace the hash value in this string with the Signature Suit's tempHash: `00FF8C6314EA882E`

```

Comment:
Just to be thorough you may want to change the TBLU hash too. This highlights the relationship between them.

You also might want to mention that they may like to format/beautify the file so it's not all on one line (better for more complex editing).